src/context.js

```diff
@@ -111,6 +111,28 @@ angular.module('hypermedia')
       }},
 
       /**
+       * Perform a HTTP PATCH request.
+       *
+       * @function
+       * @param {Resource} resource
+       * @returns a promise that is resolved to the resource
+       * @see Resource#$patchRequest
+       */
+      httpPatch: {value: function (resource, data) {
+        var self = this;
+        busyRequests += 1;
+        var request = updateHttp(resource.$patchRequest(data));
+        return $http(request).then(function () {
+          Resource.prototype.$merge.call(resource, request.data);
```

Comment:
Good point using `request.data` here and not `data` from the arguments, by the way!